### DISCUSSION TITLE
Saleor Checkout for NextJs Commerce

### DISCUSSION THREAD

**Discussion by abhi-turbo** [saleor/saleor#9733] [Q&A]
https://saleor.vercel.store/ In this demo checkout page is missing. I can see checkout has been implemented here https://github.com/vercel/commerce/pull/487 but preview is not available https://commerce-saleor-git-implement-9109fe-vercel-solutions-vtest314.vercel.app/

Plz. guide me to the checkout page implementation of Saleor NextJs commerce.

cc: @karolkielecki

**Comment by karolkielecki** [comment]
Hi @abhi-turbo, perhaps we haven't implemented the checkout page there. We started working on another project: https://github.com/saleor/react-storefront.

**Reply by abhi-turbo** [reply]
@karolkielecki Thanks for the response! I've checked the react-storefront and it's a starter-kit that requires a lot of customization. I am looking for a ready to use theme/storefront and that was the reason I was opting for NextJs Commerce. Is there any ready to use storefront available to use?

**Reply by karolkielecki** [reply]
I'm afraid there is no available ready-to-use storefront. However, we're improving the react-storefront step by step.

**Reply by abhi-turbo** [reply]
Noted! I'll try to do some contribution to the storefront over the weekend then. For now I need to figure out how to proceed.

**Comment by timuric** [comment]
@abhi-turbo we are going to release new checkout this week and it will be available in the demo of our next.js storefront https://github.com/saleor/react-storefront
stay tuned
